core/src/com/unciv/logic/simulation/Simulation.kt

```diff
@@ -30,6 +30,8 @@ class Simulation(
     var steps = ArrayList<SimulationStep>()
     var numWins = mutableMapOf<String, MutableInt>()
     private var summaryStats = HashMap<String, HashMap<Int, HashMap<String, MutableInt>>>() // [civ][turn][stat]=value
+    private val keySum = "sum"
```

Comment:
At this point, why not make this an enum?

Reply:
Yeah, good idea.